detox/src/android/actions/native.js

```diff
@@ -25,9 +25,11 @@ class TapAtPointAction extends Action {
 }
 
 class LongPressAction extends Action {
-  constructor() {
+  constructor(duration, x, y) {
```

Comment:
Wouldn't it be nicer to keep arguments as an object here `{ duration, x, y}`? This is regardless of what is chosen for the public user-facing API – I guess that transfering it to the native side should be much more pleasant, then.

Reply:
(point, duration) fits better here. changed